mlflow/keras.py

```diff
@@ -1,3 +1,4 @@
+# Revert this
```

Comment:
Added this line to trigger the keras tests. I'll revert this once I confrim setting `SPARK_LOCAL_IP` make them stable.

Reply:
Confirmed that they are stable now:

- https://github.com/mlflow/mlflow/pull/4237/commits/57bf01490da6d3ab285a8604ee8ef4067e49a76d
- https://github.com/mlflow/mlflow/pull/4237/commits/8a0fc396de9c551c5047144b02246093b6938d05
- https://github.com/mlflow/mlflow/pull/4237/commits/b50143b770a2a5b1d8ad096ad656daeffd574240